ceph-releases/luminous/ubuntu/16.04/daemon/osd_scenarios/osd_disk_activate.sh

```diff
@@ -61,7 +61,7 @@ function osd_activate {
 
     # Open LUKS device(s) if necessary
     if [[ ! -e /dev/mapper/"${DATA_UUID}" ]]; then
-      open_encrypted_part "${BLOCK_UUID}" "${DATA_PART}" "${BLOCK_UUID}"
+      open_encrypted_part "${BLOCK_UUID}" "${DATA_PART}" "${DATA_UUID}"
     fi
     if [[ ! -e /dev/mapper/"${BLOCK_DB_UUID}" ]]; then
       open_encrypted_part "${BLOCK_DB_UUID}" "${BLOCK_DB_PART}" "${BLOCK_UUID}"
```

Comment:
This needs to change as well and this line too: https://github.com/ceph/ceph-docker/pull/773/files#diff-1a01a84c16b7e013a70a97a6f473460cR70